CSS files/components/buttons/buttons.css

```diff
@@ -1,16 +1,18 @@
 
 .btn{
-    padding:0.8rem 1.2rem;
+    padding:8px 1rem;
     border-radius: .8rem;
     border:none;
     cursor: pointer;
+    font-size: 1.2rem;
 }
 
 .btn-large{
-    padding:8px 1.4rem;
-    border-radius: 15px;
+    padding:10px 1.2rem;
+    border-radius: 8px;
     border:none;
```

Comment:
You are repeating properties. `border: none` and `cursor:pointer` are already defined on `.btn` class. Chain the `.btn` class wherever you are using `btn-large`.

Reply:
Okay got it. Thank you for your valuable feedback.